apollo-runtime/src/main/java/com/apollographql/apollo/interceptor/ApolloInterceptor.java

```diff
@@ -54,6 +54,13 @@ void interceptAsync(@Nonnull InterceptorRequest request, @Nonnull ApolloIntercep
     void onResponse(@Nonnull InterceptorResponse response);
 
     /**
+     * Called when interceptor starts fetching response from source type
+     *
+     * @param sourceType type of source been used to fetch response from
+     */
+    void onFetch(FetchSourceType sourceType);
```

Comment:
What's the difference between this and FETCH_NETWORK/ FETCH_CACHE notifications?

Reply:
I guess I'm confused because we already include this data in the `Response` object as a boolean flag, so the StatusEvent callback seems sufficient.